src/common/sidebar/components/Logs.tsx

```diff
@@ -1,7 +1,15 @@
-import { Box } from "@chakra-ui/layout";
+import { Box, Heading, HStack } from "@chakra-ui/layout";
 
-function Logs() {
-  return <Box h="100%" w="100%" bg="green" />;
+interface LogProps{
```

Comment:
Just a small thing, usually I like to have the interface with the same name as component, so in this case `LogsProps`